config/install/image.style.stanford_circle.yml

```diff
@@ -0,0 +1,34 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - focal_point
+    - image_effects
+name: stanford_circle
+label: Circle
+effects:
+  0b2a1b5b-65aa-4c03-ad55-f0d64e9414d1:
+    uuid: 0b2a1b5b-65aa-4c03-ad55-f0d64e9414d1
+    id: focal_point_scale_and_crop
+    weight: -10
+    data:
+      width: 140
+      height: 140
+      crop_type: focal_point
+  716304f7-4bc8-4c20-b3da-a8f0e50ee7dd:
+    uuid: 716304f7-4bc8-4c20-b3da-a8f0e50ee7dd
+    id: image_effects_mask
+    weight: -8
+    data:
+      mask_image: modules/custom/stanford_image_styles/img/mask-image.png
```

Comment:
🤔 This might not always be true. Where this module gets installed to is dependant on what your composer settings are installed at. This is the conventional path but not guaranteed. How much effort would it be to make this more dynamic?

Reply:
yeah i thought the same. maybe just a `hook_entity_presave` could do the trick. I dont want to do a config override for this.